docs/excel/excel-add-ins-ranges-precedents-dependents.md

```diff
@@ -12,40 +12,57 @@ Excel formulas often refer to other cells. These cross-cell references are known
 
 A cell may have a precedent cell, and that precedent cell may have its own precedent cells. A "direct precedent" is the first preceding group of cells in this sequence, similar to the concept of parents in a parent-child relationship. A "direct dependent" is the first dependent group of cells in a sequence, similar to children in a parent-child relationship. Cells that refer to other cells in a workbook, but whose relationship is not a parent-child relationship, are not direct dependents or direct precedents.
```

Comment:
Consider removing or changing the last sentence of this paragraph. Now that the article has expanded beyond direct dependents and precedents, that sentence makes it seem like things outside of the parent-child relationship aren't relevant here, and that's not true any more.

Reply:
I'm glad you noticed this. I tried multiple variations of this sentence and didn't find a good fit. I think the sentence is valuable in that it reiterates what a direct dependent or precedent is, which is an abstract concept. What do you think of this version? 
> Cells that refer to other cells in a workbook, but whose relationship is not a parent-child relationship, are dependents or precedents, but they are not direct dependents or direct precedents.

Reply:
Maybe the problem is we're not defining the more "complex" relationship first. How about something like this?
```suggestion
A precedent cell may have its own precedent cells. Every precedent cell in this chain of precedents is still a precedent of the original cell. The same relationship exists for dependents: any cell affected by another cell is a dependent of that other cell. A "direct precedent" is the first preceding group of cells in this sequence, similar to the concept of parents in a parent-child relationship.  A "direct dependent" is the first dependent group of cells in a sequence, similar to children in a parent-child relationship.
```